app/views/checklist_templates/_checklist_template.html.erb

```diff
@@ -0,0 +1,22 @@
+<div id="<%= dom_id checklist_template %>">
+  <p>
+    <strong>Name:</strong>
+    <%= checklist_template.name %>
+  </p>
+
+  <p>
+    <strong>String:</strong>
+    <%= checklist_template.string %>
```

Comment:
Oh maybe I was incorrect above. Is `string` a field here? Same question below for `text`.